libs/shared/db/mysql/account/project.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "shared-db-mysql-account",
+  "$schema": "../../../../../node_modules/nx/schemas/project-schema.json",
+  "sourceRoot": "libs/shared/db/mysql/account/src",
+  "projectType": "library",
+  "targets": {
```

Comment:
Why no build target?

Reply:
Ah, mainly as I hadn't gotten here yet to update it to include one. Was fixing the core to handle import issues first.